src/main/java/de/srendi/advancedperipherals/common/util/LuaConverter.java

```diff
@@ -405,7 +384,8 @@ public static Object effectToObject(MobEffectInstance effect) {
         return map;
     }
 
-    private static Map<Integer, Object> entityToLuaUUID(Entity entity) {
-        return ObjectUtil.nullableValue(entity, e -> LuaConverter.uuidToLua(e.getUUID()));
+    @Nullable
+    private static String entityToUUIDString(@Nullable Entity entity) {
+        return ObjectUtil.nullableValue(entity, e -> e.getUUID().toString());
```

Comment:
I'm just interested why don't just do `a == null ? null : a.xxx` but pass a callback instead?

Reply:
Uhhhh good question. A little bit of overengineering I think. I'll change those